Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Tables"
 uuid = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 authors = ["quinnj <quinn.jacobd@gmail.com>"]
-version = "1.10.1"
```

Comment:
Also DataAPI.jl version requirement needs to be changed below.

Reply:
Ho yes I forgot this you're right.